zebra-chain/src/fmt/time.rs

```diff
@@ -42,3 +47,17 @@ pub fn humantime_milliseconds(duration: impl Into<Duration>) -> String {
 
     format!("{duration}")
 }
+
+/// Returns a human-friendly formatted string for the whole number of milliseconds in `duration`.
```

Comment:
```suggestion
/// Returns a human-friendly formatted string for the whole number of microseconds in `duration`.
```

(These changes were made while checking for places where `spawn_blocking()` may have been needed since `StateService::new()` has been made into an async method)